### DISCUSSION TITLE
`nuxt generate` Does Not Generate Transformed Images

### DISCUSSION THREAD

**Discussion by piet-maier** [vercel/community#1661] [Help]
### Summary

I already opened [an issue on the nuxt/image repository](https://github.com/nuxt/image/issues/757), but since this problem only occurs when deploying to Vercel, and not when building locally, I don't know if it is a problem with the module or with Vercel.

What I have is a Nuxt 3 project using the Image module. When using `nuxt build` everything works normally. But when using `nuxt generate` to instead generate a static website, on Vercel, the transformed image URLs (e.g. `my-domain.com/_vercel/image?url=/images/example.jpg&w=1280&q=100`) all return a 404 error, meaning any image that uses these transformations cannot be found. The "original" images, which are located in the `/public` folder and served under their file path inside this folder (e.g. `/public/example.jpg` can be found under `my-domain.com/example.jpg`) still work, but since I used `sizes` and `srcset` everywhere, none of my images work.

**Comment by amyegan** [comment]
Hi @piet-maier! Are you using [`@nuxtjs/vercel-builder`](https://github.com/nuxt/vercel-builder) to [deploy Nuxt with Vercel](https://nuxtjs.org/deployments/vercel/)? I haven't worked with Nuxt much, but if you can share a minimal reproduction I can try to help with debugging

**Reply by piet-maier** [reply]
> This is a legacy builder and only works for Nuxt 2.

> If you want to deploy a Nuxt Bridge or Nuxt 3 app, Vercel deployment will work with zero configuration.


Reproduction: [GitHub Repository](https://github.com/piet-maier/vercel-image-bug) & [Deployment](https://vercel-image-bug.vercel.app/)

**Reply by amyegan** [reply]
Thanks for sharing this example, Piet. I tried a few different ways and the problem seems to be replicable on other platforms, so this is likely a Nuxt bug. The next step would be to open a Nuxt issue, which you already have done 🙇
